project/Dependencies.scala

```diff
@@ -0,0 +1,24 @@
+import sbt._
+
+object Version {
+
+  val finchx       = "0.32.1"
+  val finchxCirce  = "0.32.1"
```

Comment:
Maybe we can have just the finch and circe versions, instead of each package, as they are going to evolve in the same way. WDYT?

Reply:
👌